### DISCUSSION TITLE
Cannot view related records on the time warp except Cases!

### DISCUSSION THREAD

**Discussion by Mariiakq** [deejay-hub/timeline-lwc#216] [Q&A] [answered]
Hello, 

I'm having an issue with the time wrap where it's only displaying cases and non of the other related records (evets, tasks, opportunities..etc) are showing. The time wrap is added to the Account record

I have checked the filter is selected for all related objects.
I have also checked the metadata type for both events and tasks to check the Fallback Tooltip Field and I can see Status and Owner.Name so I assume the Fallback Tooltip Field is not .. 

Could you please help me resolve this issue? I'm not sure what else I should check? 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/110628712/193570616-5e253d8f-2a87-42a2-b036-9ca923aac1c0.png">

**Comment by deejay-hub** [comment, marked as answer]
I have included a section in the FAQ document that may help (https://quip.com/6yvoAcBukqZB).

See the section called **_What about indirect relationships? The standard Account timeline shows tasks and events not directly linked to the Account. Can Time Warp do something similar?_**

What I suspect you have are Events, Tasks, Calls and Emails that are not linked directly to the Account record but are indirect relationships. Follow the instructions in the Knowledge Base document and it should help resolve your issue.

In short. You need to inactivate Tasks and Events for the Account object and activate OpenActivities and ActivityHistory relationships. The FAQ doc has more detailed instructions including screenshots.

**Reply by vinodreddys** [reply]
@deejay-hub  Is it possible to apply same for Indirect relationships on custom objects?.

**Reply by deejay-hub** [reply]
Hi @vinodreddys. No the code base doesn't support indirect relationships on custom objects. We aim to plot records you can see on the standard Activity Timeline + direct relationships to other standard and custom objects. Other use cases would have to carefully consider scale and performance. People are more than welcome to modify the code base to suit their specific needs if needed.

**Comment by Mariiakq** [comment]
Hi @deejay-hub ,

Thank you the trick has worked for the Account but not for the Person Account? and I cannot see a similar name for the person account (i.e. Person Account -OpenActivities , Person Account - ActivityHistory)

**Comment by deejay-hub** [comment]
Hi @Mariiakq it's hard to visualise what you're expecting here. Would you be able to give us the steps needed to replicate the issue you're seeing with some test data in a sandbox? Person Accounts can be related to a Task and Event in multiple ways. You also have the option to enable multiple contacts to activities. It's not clear what your setup is.

If you're unable to do so but think that having OpenActivities and ActivityHistory relationships for Person Accounts would solve your issue then you can just add the 2 records needed to the Timeline_Configuration__mdt. Under Setup->Custom Metadata Types.

We don't include records for every possibility as it uses your storage. Here is a record I created to plot Open Activities for Person Account. Note that you'll have to deactivate the following records
[Person_Tasks_Account]
[Person_Tasks_Contact]
[Person_Events_Account]
[Person_Events_Contact]

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/7021883/193706421-94075b84-036c-491c-a03c-78bdb3db8a95.png">

This then shows indirect relationships. In this case a task against a related opportunity

<img width="1456" alt="image" src="https://user-images.githubusercontent.com/7021883/193706992-e103e4f3-21d9-4ca0-af53-f4224d378dd4.png">

**Reply by deejay-hub** [reply]
That way people can plot their own custom objects too.
